go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	github.com/onflow/flow-core-contracts/lib/go/contracts v0.7.1
 	github.com/onflow/flow-go-sdk v0.15.0
 	github.com/onflow/flow-go/crypto v0.12.0
-	github.com/onflow/flow/protobuf/go/flow v0.1.9
+	github.com/onflow/flow/protobuf/go/flow v0.2.0
```

Comment:
new tag https://github.com/onflow/flow/releases/tag/protobuf%2Fgo%2Fflow%2Fv0.2.0 to include the change on master which adds the `BlockID` to `TransactionResult`